### ISSUE TITLE
CMD/ENTRYPOINT and WORKDIR is not set

### ISSUE THREAD

**Issue by moqmar** [abhin4v/hastatic#3]
It would make it much easier to use hastatic without a custom Dockerfile if the following lines were added:

```
WORKDIR "/var/www"
ENTRYPOINT ["/usr/bin/hastatic"]
```

**Comment by moqmar** [comment]
I decided to also set the port to 80, as it's the default in basically all reverse proxies and also makes sense when directly exposing the container.

**Comment by beantaxi** [comment]
@moqmar I actually came to Issues to suggest adding exactly these features. Do you have a fork, or some way of using them? If not, I can just 'borrow' your solution from your commit. Thanks!

**Comment by moqmar** [comment]
No, Haskell isn't something I know or have wanted to learn, so I actually ended up rewriting [something similar in Go](https://codeberg.org/momar/web). It doesn't support HTTPS (as using Docker implies a reverse proxy in most cases), but is quite a bit more configurable than hastatic, and even a bit faster.

**Comment by beantaxi** [comment]
@moqmar I just fired up web/momar! It works great, but I do have an issue, and I can't raise issues without joining codeberg, which I'd rather not do ...

I was hoping that web/momar would allow disabling CORS, so I can host any type of content, eg font files, without a CORS warning. From a quick look at the code those headers appear hardcoded. I like your choice of ignoring HTTPS, which is unnecessary for simple local testing, and I was wondering if you might feel the same about disabling security headers, at least optionally.

Thanks!

**Comment by moqmar** [comment]
I agree that that'd be useful & copied your request to [the Codeberg repo, together with a suggestion on security headers in general](https://codeberg.org/momar/web/issues/1) and can probably look into that tomorrow.

TL;DR: only the `Referrer-Policy: strict-origin-when-cross-origin` header will stay, and there will be an `ENABLE_CORS` variable that allows hosting web fonts and accessing static content via JavaScript, which will be disabled by default.

Let's not spam this issue here any further, just check back on Thursday, I'll probably have an update ready then. For further questions you can also use my email address in my profile here on GitHub.
